benchmark/benchmain/main.go

```diff
@@ -243,6 +246,38 @@ func makeFuncStream(benchFeatures stats.Features) (func(int), func()) {
 	}, cleanup
 }
 
+func makeFuncUnconstrainedStreamPreloaded(benchFeatures stats.Features) (func(int), func(int), func()) {
+	tc, cleanup := makeClient(benchFeatures)
+
+	streams := make([]testpb.BenchmarkService_StreamingCallClient, benchFeatures.MaxConcurrentCalls)
+	for i := 0; i < benchFeatures.MaxConcurrentCalls; i++ {
+		stream, err := tc.UnconstrainedStreamingCall(context.Background())
+		if err != nil {
+			grpclog.Fatalf("%v.UnconstrainedStreamingCall(_) = _, %v", tc, err)
+		}
+		streams[i] = stream
+	}
+
+	pl := bm.NewPayload(testpb.PayloadType_COMPRESSABLE, benchFeatures.ReqSizeBytes)
+	req := &testpb.SimpleRequest{
+		ResponseType: pl.Type,
+		ResponseSize: int32(benchFeatures.RespSizeBytes),
+		Payload:      pl,
+	}
+
```

Comment:
This function is same with `makeFuncUnconstrainedStream` until this point. Can we factor out this common part into a new function and make this one and `makeFuncUnconstrainedStream` use that new function?

Reply:
Easily done.